src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -288,6 +292,7 @@ def basic(
             ContainerType.coverage,
             ContainerType.unique_inputs,
             ContainerType.regression_reports,
+            ContainerType.readonly_inputs
```

Comment:
This will create a `readonly_inputs` container for you, which I don't think is what you want.  You should be ok removing this here.

Reply:
Is it fine if a readonly container isn't defined when we run:

fuzzer_containers = [
            (ContainerType.setup, containers[ContainerType.setup]),
            (ContainerType.crashes, containers[ContainerType.crashes]),
            (ContainerType.inputs, containers[ContainerType.inputs]),
            (ContainerType.readonly_inputs, containers[ContainerType.readonly_inputs]),
        ]

Reply:
Instead of always it to fuzzer_containers, you could instead only add it to fuzzer_containers if the user specifies it.